compose/testcontainers/compose/__init__.py

```diff
@@ -57,6 +61,8 @@ def __init__(
         self.build = build
         self.env_file = env_file
         self.services = services
+        self._user_defined_compose_command = compose_command.split(" ") if compose_command else None
+        self.docker_compose = self.base_docker_compose()
```

Comment:
Can we clean up the naming conventions here? We currently have

- `_user_defined_compose_command` as a private attribute.
- `compose_command` as a parameter of the constructor.
- `docker_compose` as a public attribute.
- `base_docker_compose` as a public function.

Can we consolidate?

Reply:
dropped `_user_defined_compose_command`, made `base_docker_compose` a private function (and renamed), and renamed `docker_compose` attribute to `compose_command` to mirror the constructor